neonvm/runner/main.go

```diff
@@ -359,19 +417,145 @@ func calcDirUsage(dirPath string) (int64, error) {
 	return size, nil
 }
 
-func createSwap(diskName string, diskPath string, diskSize *resource.Quantity) error {
-	if diskSize == nil {
-		return errors.New("diskSize should be specified")
+const (
+	swapDiskClusterSize string = "2M"
+	// Note: we use an offset of 2MiB here in order to match the cluster_size=2M that we give to
+	// qemu-img convert when creating the qcow2 image.
+	// Does it really matter? probably not? But it takes only an additional 1MiB from the
+	// partition, so IMO it's worth the peace of mind.
+	swapStartLocationMiB int64 = 2
+	swapAlignMiB         int64 = 2
```

Comment:
nit: I'd do this https://go.dev/play/p/-rLyXg4_tWV

Reply:
Fair enough, yeah - done